src/lib.rs

```diff
@@ -38,15 +38,12 @@
     html_logo_url = "https://raw.githubusercontent.com/smol-rs/smol/master/assets/images/logo_fullsize_transparent.png"
 )]
 
-#[cfg(not(all(loom, feature = "loom")))]
 use std::sync;
 
-#[cfg(all(loom, feature = "loom"))]
-use loom::sync;
-
```

Comment:
Instead of getting rid of `loom` imports entirely, can you modify [this line](https://github.com/smol-rs/parking/blob/429bb0b72aa59f5e4e369b4449fd0b6b37f4ab7d/src/lib.rs#L54) so that we import `Arc` from `std::sync` instead of `sync`?